test/linter/test-browsers-data.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+const path = require('path');
+const chalk = require('chalk');
+const { Logger } = require('./utils.js');
+
+/**
+ * @typedef {import('../../types').Identifier} Identifier
+ * @typedef {import('../utils').Logger} Logger
+ */
+
+/**
+ * @param {Identifier} data
+ * @param {Logger} logger
+ * @returns {boolean}
+ */
+function processData(data, logger) {
+  const browser = Object.keys(data.browsers)[0];
+  const releases = data.browsers[browser].releases;
+
+  let releaseByStatus = {
+    current: null,
+    beta: null,
+    nightly: null,
+    esr: null,
+  };
+
+  for (let releaseVersion in releases) {
+    const releaseData = releases[releaseVersion];
+
+    if (
+      (browser == 'nodejs'
+        ? ['beta', 'nightly']
+        : ['current', 'beta', 'nightly', 'esr']
```

Comment:
Does this mean we allow multiple `current` on Node.js?

Reply:
Based upon the linter, we do, however I realize that NodeJS wouldn't have multiple `current` releases, only multiple `esr` releases.  Lemme fix that up real quick...

Reply:
Looking at the Firefox ESR release cycle, It seems Firefox can also have multiple ESR releases for a short time. https://support.mozilla.org/en-US/kb/firefox-esr-release-cycle